pkg/storage/sqlstorage/pagination.go

```diff
@@ -14,6 +14,7 @@ type TxsPaginationToken struct {
 	DestinationFilter string    `json:"destination,omitempty"`
 	StartTime         time.Time `json:"start_time,omitempty"`
 	EndTime           time.Time `json:"end_time,omitempty"`
+	Limit             uint      `json:"limit,omitempty"`
```

Comment:
Ok, i let Limit on pkg/storage package as 'limit' make sense in sql terminology. But since the storage is not supposed to be bound on sql implementation, it make sense to change to pageSize everywhere.